ash/src/lib.rs

```diff
@@ -40,6 +42,10 @@
 //! * **loaded** (default): Support searching for the Vulkan loader manually at runtime.
 //! * **linked**: Link the Vulkan loader at compile time.
 
+#[cfg(not(feature = "std"))]
+#[macro_use]
+extern crate alloc;
```

Comment:
Are allocations something that we might later want to make entirely optional (separate change/PR)?

Reply:
Possibly.

This would however require radically changing how ash approaches the two call pattern in Vulkan unless we want to have those not using alloc call the raw vulkan functions themselves. It's a much larger discussion that needs it's own issue.

For now I required alloc so the same ash api works with no_std

Reply:
Not now, but we could do it later. You are right that the two-call-pattern becomes increasingly complex when we do this, effectively having to split all those functions in two.